torch_xla/csrc/tensor_util.cpp

```diff
@@ -925,7 +925,7 @@ std::vector<torch::lazy::BackendDataPtr> CreateTensorsData(
       // host communications. This means that we may need to manually shard
       // across global devices for multi-host training.
       std::vector<std::string> local_devices =
-          xla::ComputationClient::Get()->GetLocalDevices();
+          xla::ComputationClient::Get()->GetAllDevices();
```

Comment:
why this change?

Reply:
It's the same on the single host, but on the pod -- I need to list all the devices available.